x/evm/types/msg_vote_chain.go

```diff
@@ -37,11 +37,16 @@ func (m VoteConfirmChainRequest) ValidateBasic() error {
 	if err := sdk.VerifyAddressFormat(m.Sender); err != nil {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, sdkerrors.Wrap(err, "sender").Error())
 	}
+
 	if err := utils.ValidateString(m.Name); err != nil {
 		return sdkerrors.Wrap(err, "invalid chain name")
 	}
 
-	return m.PollKey.Validate()
+	if err := m.PollKey.Validate(); err != nil {
```

Comment:
Why?

Reply:
Just consistency, avoiding duplicated checks like the one I found.